src/server/src/main/java/io/cassandrareaper/storage/MemoryStorageFacade.java

```diff
@@ -127,4 +178,126 @@ public IClusterDao getClusterDao() {
     return this.memClusterDao;
   }
 
-}
\ No newline at end of file
+  private void persist(Object... objects) {
+    synchronized (memoryStorageRoot) {
+      try {
+        this.embeddedStorage.storeAll(objects);
+        this.embeddedStorage.storeRoot();
+      } catch (Throwable ex) {
```

Comment:
**Issue:** Can we catch a `RuntimeException` instead of a `Throwable` which is a bit too wide?

Reply:
This was put in to debug where/what was causing the not-persistable issue I was facing. I can probably just remove the try-catch all together, unless you think it would be helpful to catch it?